pkgs/applications/audio/ashuffle/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchgit, cmake, git, pkg-config, mpd_clientlib, meson, ninja }:
+
+stdenv.mkDerivation rec {
+  pname = "ashuffle";
+  version = "3.4.0";
+
+  src = fetchgit {
```

Comment:
is it possible to use `fetchFromGitHub` instead?

Reply:
Good point, I did not see the `fetchSubmodules` option.